packages/framework/src/Model/CombinedArticle/CombinedArticleElasticsearchRepository.php

```diff
@@ -37,15 +37,29 @@ public function getArticlesBySearchText(string $searchText, ?int $limit = null):
         return $this->extractHits($result);
     }
 
+    /**
+     * @param int $domainId
+     * @param int $from
+     * @param int $maxResults
+     * @return array
+     */
+    public function getAllArticles(int $domainId, int $from, int $maxResults): array
```

Comment:
Shouldn't be this method named `getArticlesByDomain(Id)`?

Reply:
nice idea, renamed in 1b3efeb77e